src/class-wpml-gutenberg-integration.php

```diff
@@ -121,7 +121,7 @@ public function string_translated(
 			}
 
 			$this->sitepress->switch_lang( $lang );
-			wp_update_post( array( 'ID' => $translated_post_id, 'post_content' => $content ) );
+			wp_update_post( array( 'ID' => $translated_post_id, 'post_content' => $content );
```

Comment:
@kkarpieszuk, you also removed a closing parenthesis with the extra space. I think this is why our tests are failing now.

Reply:
ups, fixed. @strategio please review